infrastructure/templates/public-api/components/postgresqlDatabase.bicep

```diff
@@ -72,8 +72,8 @@ param firewallRules {
 param tagValues object
 
 var databaseServerName = empty(serverName)
-  ? '${resourcePrefix}-psql-server2'
-  : '${resourcePrefix}-psql-server2-${serverName}'
+  ? '${resourcePrefix}-psql-flexibleserver'
+  : '${resourcePrefix}-psql-flexibleserver-${serverName}'
```

Comment:
It's because I created the resources in a different Resource Group / Subscription (the Hive ones).

This is some safety to prevent subdomain hijacking - it effectively stops someone from stealing your resource names from under you if you've just temporarily removed them.

Reply:
After 30 days, assuming you haven't recreated or undeleted the resource, it'll be purged and other people / Resource Groups / Subscriptions can take the name of the resource.